src/app/layout.tsx

```diff
@@ -74,6 +74,21 @@ export default async function RootLayout({
       ui?.favicon?.["shortcut-icon"] || defaultFavicons["shortcut-icon"],
   };
 
+  const bannerStyle: React.CSSProperties = {
```

Comment:
This banner makes my eyes bleed lololo

Reply:
Thats what I going for. Eye-bleeding-red.